config/boot.rb

```diff
@@ -6,3 +6,11 @@
 
 # Load configuration early
 require_relative "load_config"
+
+# Ruby 3.2 removed the `exists?` alias from `File`, but at least the `logging-rails` gem still uses it.
+# This is only a workaround, and we need a different long-term solution. It looks like the `logging-rails` gem
+# is not maintained anymore, so we maybe need to find a replacement. But since this is the only thing preventing
+# us from upgrading to Ruby >= 3.2, we can just create our own alias for now.
+class << File
```

Comment:
I wonder if we should have a dedicated `monkey_patches.rb` file just to make this a bit more discoverable. But I'm fine with this as-is.

Reply:
Yeah, I was wondering that as well, but decided against it (as I hope we don't need a lot of monkey patches and a dedicated file would maybe more encourage it? :man_shrugging:). But if we need more, we could create one.

Reply:
Sounds good, let's keep it as-is :)